tools/cldr-apps/js/src/esm/cldrInfo.mjs

```diff
@@ -452,6 +455,19 @@ function addForumPanel(tr) {
   }
 }
 
+/**
+ * An empty paragraph at the bottom of the info panel enables scrolling
+ * to bring the bottom content fully into view without being overlapped
+ * by the xpath shown by addXpath
+ */
+function addBottom() {
```

Comment:
should it be `padding-bottom: 2em;` or something in the CSS for the element instead? that would have a similar result without requiring mutation of the html

Reply:
This is quick and avoids going deep into the legacy css. The order of elements in the info panel seems likely to change. Having this new "bottom" element at the bottom keeps its style separate from the rest.